environment.yml

```diff
@@ -0,0 +1,16 @@
+channels:
+    - conda-forge
+    - defaults
+
+dependencies:
+    - python
+    - setuptools
+    - pip
+    - numpy
+    - scipy
+    - pyyaml
```

Comment:
Do we require `ruamel_yaml` with the changes to string quoting?

Reply:
This is taken care of in #44